### ISSUE TITLE
The space before the "-" is missing when using CSS variables

### ISSUE THREAD

**Issue by Eoyo** [Igorbek/typescript-plugin-styled-components#233]
```ts
TypeScript before transform:

  declare const styled: any;
  const ValueCalc = styled.div \`
    height: calc(var(--line-height) - 5px);
    width: calc(100% - 5px);
  \`;
  

TypeScript after transform:

  declare const styled: any;
  const ValueCalc = styled.div \`height:calc(var(--line-height)- 5px);width:calc(100% - 5px);\`;
  
```
There is no problem when using only the `calc` method, but there is a bug when combined with the `var` method. Browser can't parse missing spaces before '-'

**Comment by Igorbek** [comment]
@Eoyo 
```ts
  const ValueCalc = styled.div \`height:calc(var(--line-height)- 5px);width:calc(100% - 5px);\`;
//                                                             ^ is the space missing in this position?
```

**Comment by Sealarn** [comment]
I have the same issue.
```
 --padding-button: calc(var(--padding-button-vertical)- 2px) calc(var(--padding-button-horizontal)- 2px);
                                                      ^                                           ^
```

**Comment by Igorbek** [comment]
@Eoyo @Sealarn it's now been fixed and released in 2.0.0, see the announcement https://github.com/Igorbek/typescript-plugin-styled-components/discussions/440
